packages/use-contractkit/src/connectors/connectors.ts

```diff
@@ -37,14 +37,21 @@ export class UnauthenticatedConnector implements Connector {
   public initialised = true;
   public type = WalletTypes.Unauthenticated;
   public kit: ContractKit;
-  public account: string | null = null;
+  public account: Maybe<string> = null;
   public feeCurrency: CeloTokenContract = CeloContract.GoldToken;
   constructor(n: Network) {
     this.kit = newKit(n.rpcUrl);
   }
 
+  persist() {
```

Comment:
This feels like a strange name for what it's doing.

Reply:
resolving this to continue discussion in https://github.com/celo-org/use-contractkit/pull/169/files/7e67af444d4edb8ea9823bcfe2371beefa9b15e9#diff-207ee378b1d3ae745e38a1fffaf4783da7b2531ec66e0c5e09b05b4ae5adaa5d